docs/Getting Started.md

```diff
@@ -0,0 +1,778 @@
+# Getting Started
+
+This document introduces the basic steps needed to install all components of cloudflow and deploy and run a simple cloudflow application on the GKE cluster.
+
+We will discuss the following steps in sequence:
+
+1. Download and install cloudflow CLI
+2. Go through all steps of developing a sample hello world application using cloudflow libraries
+3. Install a GKE cluster following instructions in `cloudflow-installer` repository
+4. Build the application and publish to GKE repository using `sbt buildAndPublish`
+5. Deploy the application to the cluster using cloudflow CLI
+6. Show usage of CLI helpers like `status`, `list` etc.
+7. Show how to deal with http ingress and push some data to the application
+8. Verify that the application works
+9. Undeploy the application
+
+
+## Download and install the CLI
+
+cloudflow CLI can be downloaded from [lightbend public bintray repository](https://bintray.com/lightbend/cloudflow-cli). Download the version appropriate for your platform and install it in your local system. Please make sure you have the executable in your path with proper permission settings.
```

Comment:
s/cloudflow/Cloudflow

Reply:
Should we provide some cli screenshots or use something like: https://github.com/faressoft/terminalizer

Reply:
This is just for the download part. I am assuming we will have a separate guide for the CLI ..